src/lamplib/src/genny/tasks/preprocess.py

```diff
@@ -0,0 +1,334 @@
+import os
```

Comment:
This is mostly a line-by-line translation of workload_parsers.cpp in the genny driver code.